src/Repository/MatchesRepository.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository;
+
+class MatchesRepository implements RepositoryInterface
+{
+    public function __construct(private readonly \PDO $connection)
+    {
+
+    }
+
+    public function insertMatches(array $matches): void
+    {
+        if (count($matches) <= 0) {
```

Comment:
You could use empty https://www.php.net/manual/en/function.empty.php

__optional__ but recomended